cogboard-webapp/src/components/widgets/dialogFields/SonarQubeMetricsInput.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
 
-import { Input, InputLabel, Checkbox, MenuItem, ListItemText, Select } from '@material-ui/core';
+import { Input, InputLabel, Checkbox, MenuItem, ListItemText, Select, FormHelperText } from '@material-ui/core';
 
 import { StyledFormControl } from './../../styled';
 import {ALL_SONARQUBE_METRICS} from "../../../constants";
 
-const SonarQubeMetricsInput = props => {
-  const { onChange, value } = props;
+const SonarQubeMetricsInput = ({ onChange, value, error }) => {
   const inputId = 'sonarqube-metrics-input';
+  const hasError = error !== undefined;
```

Comment:
You mix the approaches to check if error is not undefined. i suggest to create a helper function hasError and use it in every field for consistance.

Reply:
done